ytest/response.go

```diff
@@ -17,19 +17,24 @@
 package ytest
 
 import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
 	"log"
 	"net/http"
 )
 
 type Response struct {
 	code   int
 	header http.Header
+	body   io.Reader
```

Comment:
can't meet requirement if use `io.Reader` (can't call `match` multiple times)

Reply:
And don't add too many features in a pull request.
Can create a single pr for a complex feature.

Reply:
I'm new to PR workflow. Thanks for your comments. They are significant to my future work. I'll consider adding redundant features in another PR.